projects/replicated-ship/rules.yaml

```diff
@@ -0,0 +1,34 @@
+---
+- name: spec-require-render
+  type: error
+  message: lifecycle should include at least one `render` step
+  test:
+    AllOf:
```

Comment:
I don't think I understand `AllOf` here
Do you want all lifecycles to have a render step?
Or at least one to have that?

If you're testing multiple lifecycles, you should make test cases with multiple lifecycles
If there can only be one, then we don't need AllOf - Just Exists iirc

Reply:
How would you write a rule that "at least one of the lifecycle steps has a truthy `render` field" using Exists?

Reply:
This looks much better